composer.json

```diff
@@ -11,12 +11,12 @@
         }
     ],
     "require": {
-        "php": "^5.4 || ^7.0",
-        "symfony/options-resolver": "^2.3 || ^3.0",
+        "php": "^7.1",
+        "symfony/options-resolver": "^3.0",
```

Comment:
Can we use `^2.8 || ^3.0` ? Otherwise, people using the Symfony LTS will be blocked on older versions. 2.8 contains all the features we have in 3.0